### PR TITLE
fix fmt style for dsp

### PR DESCRIPTION
**Pull request by abhilashshetty04**

### CODE REVIEW

File: k8s/operators/src/pool/main.rs [hunk 1]
```diff
@@ -257,11 +257,17 @@ async fn main() -> anyhow::Result<()> {
         env!("CARGO_PKG_VERSION"),
     );
 
-    let fmt_style = matches.get_one::<FmtStyle>("fmt-style").unwrap();
+    let fmt_style = matches.get_one::<String>("fmt-style").unwrap().as_str();
+    let fmt_style = match fmt_style {
+        "json" => FmtStyle::Json,
+        "compact" => FmtStyle::Compact,
+        _ => FmtStyle::Pretty,
+    };
```
Comment (tiagolobocastro) [inline k8s/operators/src/pool/main.rs]: leave as it was before, and update arg with `.value_parser(clap::value_parser!(FmtStyle))`

**Review by tiagolobocastro — APPROVED** [review]

**Review by hrudaya21 — APPROVED** [review]

**Comment by abhilashshetty04** [comment]
bors merge

**Comment by abhilashshetty04** [comment]
bors merge

**Comment by abhilashshetty04** [comment]
bors try

**Review by Abhinandan-Purkait — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**